src/pages/workspaces/workspace/Data.js

```diff
@@ -407,9 +422,64 @@ const EntitiesContent = ajaxCaller(class EntitiesContent extends Component {
     ])
   }
 
+  renderBulkSelectionOptions() {
+    const { selectedEntities, totalRowCount } = this.state
+    const allSelected = _.size(selectedEntities) === totalRowCount
+    return h(Fragment, [
+      `${_.size(selectedEntities)} currently selected.`,
+      !allSelected && linkButton({
+        onClick: () => this.loadAllData()
+      }, [`Select all ${totalRowCount}`]),
+      linkButton({
+        onClick: () => this.setState({ selectedEntities: {}, bulkSelecting: false })
+      }, ['Clear selection'])
```

Comment:
I know you're still working on the styling, but is the link to `Clear Selection` supposed to show up in 2 places?
<img width="1102" alt="screen shot 2018-11-29 at 2 40 29 pm" src="https://user-images.githubusercontent.com/2395211/49247263-d5cee800-f3e4-11e8-8304-5fe85377ba20.png">

Reply:
Whoops just read your full description, so this is as expected currently.